frontend/src/Views/Export/ExportPage.tsx

```diff
@@ -1,14 +1,46 @@
 /**
- * Hello World page. Nothing Special. This is a skeleton!!!
- * @desc only shows Hello World, will implement as Designer creates mockup
- * @author @Kcaparas
+ * @desc Export page for downloading your work in nmp format
+ * @author @GDamaso
  */
 
-import React from 'react';
+import Button from '@Commons/Button/Button.tsx';
+import MainPageHeader from '@Commons/MainPageHeader/MainPageHeader.tsx';
+import MainPageFooter from '@Commons/MainPageFooter/MainPageFooter.tsx';
+import { FC } from 'react';
+import { StyledContent, StyledLandingContainer } from './ExportPage.styles.ts';
 
-const ExportPage: React.FC = () => {
-  localStorage.clear();
-  return <h2>Export Page</h2>;
+const ExportPage: FC = () => {
+  const downloadFile = () => {
+    localStorage.clear();
+    console.log('Downloading');
+  };
+
+  return (
+    <StyledLandingContainer>
+      <MainPageHeader />
+      <StyledContent>
+        <div id="dataFileHeader">
+          <h6>NMP Data File </h6>
```

Comment:
Why is this an h6? 
Your headers go from H1 straight to H6
![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/62873746/46975c12-69bd-485e-937d-c0cf59972cb3)

In investigating this I can see your app is also using H4s improperly too.
Header elements have semantic meaning, please read this. https://blog.hubspot.com/marketing/header-tags